style.css

```diff
@@ -1 +1,267 @@
-/*  Add your styling here */
+/* General rules */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-style: italic;
+    font-family: 'Times New Roman', Times, serif;
+}
+
+/* Header */
+.header {
+    width: 100%;
+    height: 25vh;
+}
+
+.header-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+.header-img img {
+    /* width: 25vw; */
+    height: 15vh;
+    margin-top: 1rem;
+    margin-left: 1rem;
+    cursor: pointer;
+}
+
+.menu-btn {
+    padding-right: 2rem;
+    padding-top: 1rem;
+    font-size: 2rem;
+    cursor: pointer;
+}
+
+.menu-btn:hover {
+    font-size: 2.5rem;
+    color: rgb(190, 110, 35);
+    transition: all 0.5s ease;
+}
+
+/* Navigation Menu */
+
+.navbar {
+    display: none;
+}
+
+.navbar-list {
+    display: flex;
+    list-style-type: none;
+    justify-content: flex-end;
+    padding-right: 5rem;
+    font-size: 1rem;
+}
+
+.navbar-list a {
+    cursor: pointer;
+    text-decoration: none;
+    color: rgb(190, 110, 35);
+    margin-left: 3rem;
+}
+
+.navbar-list a:hover {
+    font-size: 1.1rem;
+    transition: all 0.4s ease;
+}
+
+.navbar-list a {
+    padding: 0.5rem;
+    background-color: rgb(190, 110, 35);
+    color: #f2f2f2;
+    border-radius: 5px;
+    text-transform: uppercase;
+}
+
+/* Main */
+main {
+    margin: auto 0.5rem;
+}
+
+/* Main first section */
+.header-desc {
+    float: right;
+    width: 50%; 
+    font-size: 1rem;   
+}
+
+.hero-desc {
+    margin: 1rem;
+    text-align: justify;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    margin: 0 2rem 1rem;
+}
+
+.hero h1 {
+    margin-top: 1rem;
+    padding-top: 0.5rem;
+    text-align: center;
+    margin-bottom: 0.5rem;
+    border-top: 1px solid rgb(190, 110, 35);
+    width: 100%;
+}
+
+.hero-desc p {
+    padding: auto 1rem;
+}
+
+.hero img {
+    width: 100%;
```

Comment:
It looks like the images are stretched in certain window sizes. This is caused by forcing a width and height on them which don't obey the image's aspect ratio. The preferable solution to this is to use similar aspect ratio images (eg images with the same width and height) and make sure the width and height you apply respects that ratio.

A different option would be to use div elements instead of img. You could then use css `background` property to apply an image to the div and use `background-size` properties you get a few more options on how to size and position your images.